package.json

```diff
@@ -24,6 +24,9 @@
     "typescript": "^3.4.3"
   },
   "resolutions": {
+    "**/**/handlebars": "4.5.3",
+    "**/**/marked": "0.8.0",
+    "**/**/https-proxy-agent": "2.2.3",
```

Comment:
let's go ahead and bump these rather than using `resolutions` - I've just merged a bump to handlebars.